tests/b006_b008.py

```diff
@@ -7,6 +7,10 @@ def this_is_okay(value=(1, 2, 3)):
     ...
 
 
+def and_this_also(value=tuple()):
```

Comment:
Why would you ever want to say "tuple()" instead of just "()"?

Reply:
I thought the same, but it is valid, all be it slower.

Reply:
You wouldn't but I have seen code which does do this. It is a styling and performance issue rather than a B008.